tests/test_power.py

```diff
@@ -14,21 +14,13 @@ def setUp(self):
         self.robot = Robot(robotd_path="/tmp/robotd")
 
     def test_on_off(self):
+        self.power_board.message_queue.get()
```

Comment:
This isn't needed now that we've gone back the original order of having the outputs come on before the start LED, and it's causing the tests to fail. Intuition suggests you might also need a `clear_queue` between lines 22 and 24 to throw away any start LED stuff, but I'm not sure.

Reply:
I've fixed this in #66.